content/en/docs/languages/java/quickstart.md

```diff
@@ -126,19 +128,30 @@ In the same directory, open
 new method like this:
 
 ```java
+// Implementation of the gRPC service on the server-side.
 private class GreeterImpl extends GreeterGrpc.GreeterImplBase {
 
   @Override
   public void sayHello(HelloRequest req, StreamObserver<HelloReply> responseObserver) {
+    // Generate a greeting message for the original method
     HelloReply reply = HelloReply.newBuilder().setMessage("Hello " + req.getName()).build();
+
+    // Send the reply back to the client.
     responseObserver.onNext(reply);
+
+    // Notify the client that the response is complete.
     responseObserver.onCompleted();
   }
 
   @Override
   public void sayHelloAgain(HelloRequest req, StreamObserver<HelloReply> responseObserver) {
+    // Generate another greeting message for the new method.
     HelloReply reply = HelloReply.newBuilder().setMessage("Hello again " + req.getName()).build();
+
+    // Send the reply back to the client.
     responseObserver.onNext(reply);
+
+    // Notify the client that the response is complete.
```

Comment:
This doesn't directly notify the client, this is the user's server code letting the gRPC library know that no further messages will be sent to the client.

So maybe something more like: "Indicate that no further messages will be returned."

Reply:
Thank you @temawi sir for correcting me.

Reply:
I have made the changes @temawi sir, can you review it